packages/block-std/package.json

```diff
@@ -16,7 +16,8 @@
     "@blocksuite/store": "workspace:*"
   },
   "dependencies": {
-    "@blocksuite/global": "workspace:*"
+    "@blocksuite/global": "workspace:*",
+    "lib0": "^0.2.74"
```

Comment:
Not sure if this will install lib0 twice? We even use `peerDeps` to make sure yjs only installs once.